dist/profile/manifests/pkgrepo.pp

```diff
@@ -14,9 +14,27 @@
   include profile::firewall
   include profile::letsencrypt
 
-  # Needed so we can generate repodata on the machine
-  package { 'createrepo':
-    ensure => present,
+  # 'Ubuntu 20.04+ are not supported (createrepo package absent).'
+  case $facts['os']['distro']['codename'] {
+    'bionic': {
+      # Needed so we can generate repodata on the machine
+      ['createrepo','python2.7'].each |$pkg| {
+        package { $pkg:
+          ensure => present,
+        }
+      }
+
+      exec { 'Define python2.7 as the default system python':
+        require => Package['python2.7'],
+        # Last argument is the weight. Bigger weight wins
+        command => '/usr/bin/update-alternatives --install /usr/bin/python python /usr/bin/python2.7 1000',
+        unless  => '/usr/bin/python --version 2>&1 | grep --quiet "2\.7\."',
+      }
+    }
+    # TODO: add support of Ubuntu 22.04 Jammy with the createrepo-c package instead
```

Comment:
Cf comment line 17 above, what about Ubuntu 20.04? Should it be mentioned in this comment too?

Reply:
The comment here is about `createrepo-c` which is only available starting from Ubuntu 22.04.
The comment on L17 is about the "currently" used package `createrepo` which is only available on Ubuntu 18.04.

Ubuntu 20.04 is basically dead to us for this use case.

Reply:
```suggestion
    # TODO: add support of Ubuntu 22.04 Jammy with the createrepo-c package instead. Ref. https://github.com/jenkins-infra/release/issues/202.
```